taworks/taform/templates/accounts/password_reset_email.html

```diff
@@ -0,0 +1,14 @@
+{% autoescape off %}
+You're receiving this email because you requested a password reset for your user account at TA-Works.
+
+Please go to the following page and choose a new password:
+{% block reset_link %}
+{{ protocol }}://{{ domain }}{% url 'password_reset_confirm' uidb64=uid token=token %}
+{% endblock %}
+Your username, in case you've forgotten:{{ user.get_username }}
+
+Thanks for using our site!
+
+Ta Works
```

Comment:
-- for consistently sake, I think we are 'TAWorks' ?? @smwatts this was our final choice right lol?

Reply:
![screen shot 2018-02-20 at 2 31 48 pm](https://user-images.githubusercontent.com/22134649/36444983-c8925f8c-164a-11e8-9f06-0a669cc60f5c.png)

from the brochure material :)